frontend/src/components/Analysis/types.tsx

```diff
@@ -22,6 +22,8 @@ export interface ResultFineGrainedParsed {
   numbersOfSamples: number[];
   // performances[i]: performance (value/confidence) for bucket i
   performances: Performance[];
+  // Used by combo count analyses
```

Comment:
Depending on the analysis type, some fields are used and some are not in this interface. I don't think this is a good design because this interface will get more and more complex as we add more analysis types and it requires the developers to understand which field is available for what analysis. Instead, we can maybe implement a base interface and derive different interfaces/classes for different analyses. I think `parse***Features` isn't really necessary and it can be replaced with simpler code. So, let's keep it as it is for now and revisit it in the future.

Reply:
Yes, I think so too. We can do the refactor when we have >2 analyses in the future.